### PR TITLE
Update the information in the XDMF section

### PR DESCRIPTION
**Pull request by XinShen-CHN**
## Proposed changes

Some updated information in the XDMF section of the documentation. #799

## Types of changes

What types of changes does your code introduce to FESTIM?

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code refactoring
- [ x ] Documentation Update (if none of the other choices apply)
- [ ] New tests

## Checklist

- [ ] Black formatted
- [ ] Unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by RemDelaporteMathurin** [comment]
Thanks for this contribution @XinShen-CHN!

Just letting you know that you can visualise the compiled documentation [here](https://festim--801.org.readthedocs.build/en/801/).

you can also compile the documentation locally. To do so follow the Documentation guide in the Developer’s guide.

**Review by RemDelaporteMathurin — CHANGES_REQUESTED** [review]
Thanks @XinShen-CHN for that first contribution 🎉 

Here are a few remarks that would need to be addressed before merging.

File: docs/source/userguide/export_post_processing.rst [hunk 1]
```diff
@@ -62,7 +62,21 @@ The following example shows how to export the solution of a 1D problem:
...
+    :width: 400
+    :align: center
+
+- Edit the colour map and rescale the colourbar to present the proper view of results
+
+.. thumbnail:: ../images/paraview_guide_2.png
+    :width: 400
+    :align: center
+
+- Find out more information on `Paraview tutorials <https://www.paraview.org/tutorials/>`_
```
[the remark this answers is not in the document]
Comment (XinShen-CHN) [reply docs/source/userguide/export_post_processing.rst:79]: Thank you very much!

Sure, I'll modify the documentation and add the new tip.

**Review by RemDelaporteMathurin — APPROVED** [review]
Looks good to me! I'll leave this open a little while longer to give others a chance to make some comments and then we can merge this PR.

**Pull request merged**